backends/gdx-backend-lwjgl3/src/com/badlogic/gdx/backends/lwjgl3/DefaultLwjgl3Input.java

```diff
@@ -87,31 +87,10 @@ public void invoke(long window, int codepoint) {
 	};
 	
 	private GLFWScrollCallback scrollCallback = new GLFWScrollCallback() {
-		private long pauseTime = 250000000L; //250ms
-		private float scrollYRemainder;
-		private long lastScrollEventTime;
 		@Override
 		public void invoke(long window, double scrollX, double scrollY) {
 			DefaultLwjgl3Input.this.window.getGraphics().requestRendering();
-			if (scrollYRemainder > 0 && scrollY < 0 || scrollYRemainder < 0 && scrollY > 0 ||
-				TimeUtils.nanoTime() - lastScrollEventTime > pauseTime ) { 
-				// fire a scroll event immediately:
-				//  - if the scroll direction changes; 
-				//  - if the user did not move the wheel for more than 250ms
-				scrollYRemainder = 0;
-				int scrollAmount = (int)-Math.signum(scrollY);
-				eventQueue.scrolled(scrollAmount);
-				lastScrollEventTime = TimeUtils.nanoTime();
-			}
-			else {
-				scrollYRemainder += scrollY;
-				while (Math.abs(scrollYRemainder) >= 1) {
-					int scrollAmount = (int)-Math.signum(scrollY);
-					eventQueue.scrolled(scrollAmount);
-					lastScrollEventTime = TimeUtils.nanoTime();
-					scrollYRemainder += scrollAmount;
-				}
-			}
+			eventQueue.scrolled(-(float)scrollX, -(float)scrollY);
```

Comment:
Is there a specific reason you roll back your fix from #5443?

Reply:
Now that we handle properly fractional scroll events down to InputProcessor there is no need for my previous fix. It was accumulating fractional scroll events to send integer values to InputProcessor. A flood of say 0.1 values was transfered as a flood of 1 values, giving a super fast scrolling.